src/controllers/controller.bar.js

```diff
@@ -66,13 +66,10 @@ function computeMinSampleSize(scale, pixels) {
 function computeFitCategoryTraits(index, ruler, options) {
 	const thickness = options.barThickness;
 	const count = ruler.stackCount;
-	const curr = ruler.pixels[index];
-	const min = isNullOrUndef(thickness)
-		? computeMinSampleSize(ruler.scale, ruler.pixels)
-		: -1;
 	let size, ratio;
 
 	if (isNullOrUndef(thickness)) {
+		const min = isNullOrUndef(thickness) ? ruler.min : -1;
```

Comment:
@benmccann Line 71 and Line 72 reuse the judgment conditions. ``isNullOrUndef(thickness)``

Reply:
@Alanscut thank you for catching that. I've updated the code